caches_bench/caches_bench_test.go

```diff
@@ -7,7 +7,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/allegro/bigcache/v2"
+	"github.com/allergo/bigcache"
```

Comment:
`allegro`, not `allergo`. 
Btwm dropping `v2` is not a proper change, we're using Go modules and bigcache's major version is 2.

Reply:
ok i will resend the PR by fixing this

Reply:
fixed this so marking it as resolved